.travis.yml

```diff
@@ -39,19 +39,13 @@ matrix:
     - jdk: oraclejdk8
       env:
         - DESC="checkstyle and sevntu-checkstyle"
-        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify -DskipTests -DskipITs -Dpmd.skip=true -Dfindbugs.skip=true -Dspotbugs.skip=true -Dcobertura.skip=true"
+        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify -DskipTests -DskipITs -Dpmd.skip=true -Dfindbugs.skip=true -Dspotbugs.skip=true -Djacoco.skip=true"
 
-    # jacoco (oraclejdk8)
+    # jacoco and codecov (oraclejdk8)
     - jdk: oraclejdk8
       env:
-        - DESC="jacoco"
-        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify -Pno-validations -Pjacoco-check -DskipTests=false"
-
-    # cobertura and codecov (oraclejdk8)
-    - jdk: oraclejdk8
-      env:
-        - DESC="cobertura and codecov"
-        - CMD="./.ci/travis/travis.sh cobertura-check"
+        - DESC="jacoco and codecov"
+        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean site -Pno-validations -Djacoco.skip=false -DskipTests=false"
```

Comment:
looks over complicated ..... 
thee should be smth like `mvn test jacoco:report` .... but for some reason I can not make it work

Reply:
Me neither. I think the point is that JaCoCo need to make the instumentation, do the tests, then create a report. Here that almost works:
```
mvn -e clean test jacoco:instrument@default-instrument jacoco:check@default-check jacoco:report@default-report
```
but this is even worse.

Reply:
@pbludov , please resolve all other points, we will think about it later.